src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Rectangle implements Figure {
+    private String color;
+    private double height;
+    private double width;
+
+    public Rectangle(String color, double height, double width) {
+        this.color = color;
+        this.height = height;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return height * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Rectangle: color="
+                + color
+                + ", height="
+                + height
+                + ", width="
+                + width
+                + ", area="
+                + getArea());
```

Comment:
Avoid using direct `System.out.println()` calls in the `draw()` method for printing the figure's details. It's better to override `toString()` method in your class and use it to return the string representation of the object, then print that string outside of the class.